sdk/lib/opentelemetry/sdk/trace/export/multi_span_exporter.rb

```diff
@@ -23,36 +23,34 @@ def initialize(span_exporters)
           #
           # @param [Enumerable<Span>] spans the list of sampled {Span}s to be
           #   exported.
+          # @param [optional Numeric] timeout An optional timeout in seconds.
           # @return [Integer] the result of the export.
-          def export(spans)
-            @span_exporters.inject(SUCCESS) do |result_code, span_exporter|
-              merge_result_code(result_code, span_exporter.export(spans))
+          def export(spans, timeout: nil)
+            start_time = Time.now
+            results = @span_exporters.map do |span_exporter|
+              span_exporter.export(spans, timeout: Internal.maybe_timeout(timeout, start_time))
             rescue => e # rubocop:disable Style/RescueStandardError
               OpenTelemetry.logger.warn("exception raised by export - #{e}")
               FAILURE
             end
+            results.uniq.max || SUCCESS
```

Comment:
The `|| SUCCESS` applies if `@span_exporters` is empty. I'd be happy to check that and return early instead.